db/migrate/20151015085732_add_persistent_volume.rb

```diff
@@ -0,0 +1,16 @@
+class AddPersistentVolume < ActiveRecord::Migration
+  def change
+    add_column :container_volumes, :ems_ref, :string
+    add_column :container_volumes, :creation_timestamp, :timestamp
+    add_column :container_volumes, :resource_version, :string
+    add_column :container_volumes, :capacity, :string
+    add_column :container_volumes, :access_modes, :string
+    add_column :container_volumes, :reclaim_policy, :string
+    add_column :container_volumes, :status_phase, :string
+    add_column :container_volumes, :status_message, :string
+    add_column :container_volumes, :status_reason, :string
+    add_column :container_volumes, :parent_type, :string
+
+    rename_column :container_volumes, :container_group_id, :parent_id
+  end
```

Comment:
@zakiva please look at `20150522161336_add_container_entities_type.rb` because you should update the  `parent_type` for the existing volumes to `ContainerGroup`.

You probably need to split between `up` and `down`. Then add:

    class ContainerVolume < ActiveRecord::Base
      self.inheritance_column = :_type_disabled # disable STI
    end

And:

    say_with_time("Update ContainerVolume type to ContainerGroup") do
      ContainerVolume.update_all(:parent_type => "ContainerGroup")
    end

I am not sure if I got all the entities names right but that's the gist of it.